src/lib/y2partitioner/actions/go_to_device_tab.rb

```diff
@@ -46,7 +46,7 @@ def run
 
         # First, pretend the user visited the device and then the tab...
         state = UIState.instance
-        state.select_page([target_page.id])
+        state.select_page([target_page.tree_path])
```

Comment:
I think you don't need to create a hash here because `#tree_path` already returns a hash.

Reply:
Sorry, I overlooked that. You are right, #tree_path already provides the needed structure. 

Thanks for catching it!